Description: Fix default sql_connection directive
 Using sqlite by default, and removing examples which are making it impossible
 to parse.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-16

--- openstack-trove-6.0.0~b2.orig/etc/trove/trove.conf.sample
+++ openstack-trove-6.0.0~b2/etc/trove/trove.conf.sample
@@ -145,9 +145,7 @@ api_paste_config = api-paste.ini
 # SQLAlchemy connection string for the reference implementation
 # registry server. Any valid SQLAlchemy connection string is fine.
 # See: http://www.sqlalchemy.org/docs/05/reference/sqlalchemy/connections.html#sqlalchemy.create_engine
-# connection = sqlite:///trove_test.sqlite
-connection = mysql+pymysql://root:e1a2c042c828d3566d0a@localhost/trove
-#connection = postgresql://trove:trove@localhost/trove
+connection = sqlite:///var/lib/trove/sqlitedb
 
 # Period in seconds after which SQLAlchemy should reestablish its connection
 # to the database.
